.github/workflows/py-ci.yml

```diff
@@ -187,7 +187,8 @@ jobs:
             python -m pip install -e lambdas/shared
             python -m pip install -e lambdas/${{ matrix.path }}
           fi
-          python -m pip install -r lambdas/${{ matrix.path }}/requirements.txt -r lambdas/${{ matrix.path }}/test-requirements.txt
+          python -m pip install --no-deps -r lambdas/${{ matrix.path }}/requirements.txt
+          python -m pip install -r lambdas/${{ matrix.path }}/test-requirements.txt
```

Comment:
Should we not do test-requirements.txt first so that requirements.txt has the last word (in case dependencies overlap; but maybe you checked that and again in that case a note in lambdas/README.md would be nice).

Reply:
The problem is, that could easily hide a missing second-order dependency from `requirements.txt`. In general, though, `test-requirements` only contain additional requirements, without any overlap - though it's not exactly enforced anywhere.

Reply:
Wait, sorry, that doesn't make sense. It'll hide the missing dependency regardless. I guess I could switch them.

Reply:
Up to you but it does seem more correct to do test-requirements first, assuming pip does things left to right because lambdas never see any of those libs.